cli/src/main/java/org/openjdk/skara/cli/JCheckCLIVisitor.java

```diff
@@ -326,6 +326,26 @@ public void visit(IssuesTitleIssue i) {
         }
     }
 
+    @Override
+    public void visit(CopyrightFormatIssue i) {
+        if (!ignore.contains(i.check().name()) && !isLax) {
+            if (!i.filesWithCopyrightFormatIssue().isEmpty()) {
```

Comment:
No need to check empty since we are iterating over it anyway. It doesn't hurt to do it, but I think it's more readable without the extra if statement.

Reply:
Yes, fixed it